Src/Tools/DiscoveryDocPatcher/Program.cs

```diff
@@ -76,6 +77,22 @@ static void PatchDirectory(string rootPath)
 
         }
 
+        static void PatchDataLabeling(string rootPath)
+        {
+            var patcher = IfFileExists(() => Patcher.Load(Path.Combine(rootPath, "datalabeling_v1beta1.json")));
+            if (patcher is null)
+            {
+                return;
+            }
+            // Strongly-type properties that are defined without typing.
+            patcher.Replace(
```

Comment:
It might be worth copying the PR description in here to explain the purpose of it.

Reply:
Ah yes, will do, and remove that comment which is from copy-pasting.